.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -53,6 +53,10 @@ parameters:
   displayName: Build MarinerV2 Gen2
   type: boolean
   default: true
+- name: buildMarinerV2gen2ARM64
```

Comment:
Could you ask @alexeldeib to grant you write permission on AgentBaker repo, so that you can make your change on your branch on AgentBaker repo instead of your fork. Then you can test the Mariner 2.0 ARM64 build in pipeline https://msazure.visualstudio.com/CloudNativeCompute/_build?definitionId=172842

Reply:
Pipeline has been [executed ](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=58216239&view=results). There were two images which failed to build. These were both related to insufficient cores in the standardNCSv3Family Cores quota.

Additionally, [this pipeline](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=58214924&view=results) was automatically triggered when my branch was pushed to AB's repository. It appears to have succeeded, no issue.

Reply:
Two Gen2+GPU failures because of standardNCSv3Family Cores quota are expected. I approved your PR. Not sure if you can merge yourself. If you cannot, you can ask Henry in your team or me to merge on behalf.